06-variables/04-assignment/exercises/09-swapper-2/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	red, blue := "red", "blue"
+
+	red, blue = "blue", "red"
```

Comment:
it is an incorrect implementation

you should swap variables, not assign a new values